apps/kg/distributed/luanch.sh

```diff
@@ -0,0 +1,8 @@
+SCRIPT_PATH=~/dgl/apps/kg/distributed
```

Comment:
Can you write a readme for these scripts ?

Reply:
Sure.